ingest/scripts/call_loculus.py

```diff
@@ -483,7 +483,7 @@ def record_factory(*args, **kwargs):
     if mode == "get-submitted":
         logger.info("Getting submitted sequences")
         response = get_submitted(config)
-        Path(output).write_text(json.dumps(response), encoding="utf-8")
+        Path(output).write_text(json.dumps(response, indent=4, sort_keys=True), encoding="utf-8")
```

Comment:
This change is unrelated - could split it into separate PR

Just to make output diffs more stable